src/open_producten/locaties/tests/api/test_organisatie_filters.py

```diff
@@ -0,0 +1,85 @@
+from django.urls import reverse
+
+from rest_framework import status
+
+from open_producten.locaties.tests.factories import OrganisatieFactory
+from open_producten.utils.tests.cases import BaseApiTestCase
+
+
+class TestOrganisatieFilters(BaseApiTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.path = reverse("organisatie-list")
+
+    def test_naam_filter(self):
+        OrganisatieFactory.create(naam="Maykin Media")
+        OrganisatieFactory.create(naam="Gemeente A")
+
+        response = self.client.get(self.path, {"naam__iexact": "Maykin Media"})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data["count"], 1)
+
```

Comment:
This is just the way I test, it is not mandatory, but I have always tested the values of the answer as well
```suggestion
self.assertEqual(response.data["results"][0]["naam"], "Maykin Media")
```